theano/tensor/nnet/tests/test_nnet.py

```diff
@@ -287,6 +287,31 @@ def myfunc(x):
         f = theano.function([], myfunc(sa))
         self.assertTrue(check_stack_trace(f, ops_to_check='all'))
 
+    def test_logsoftmax_grad_true_div_elemwise(self):
+        """
+        Checks that the gradient of an expression similar to a log(softmax) but
+        with a different elemwise operation than true_div is not optimized.
+        """
```

Comment:
Can you put that documentation in a comment rather than in the docstring? That way, nosetests will not print the first line of the docstring instead of the test name.